x-pack/docs/en/rest-api/security/saml-authenticate-api.asciidoc

```diff
@@ -0,0 +1,86 @@
+[role="xpack"]
+[[security-api-saml-authenticate]]
+=== SAML authenticate API
+
+Submits a SAML response message to {es} for consumption.
```

Comment:
I think it's worth having the introductory _when using a custom web application other than Kibana_ sort of text in here in case people land on the page from a search result.